src/main/java/teammates/common/datatransfer/FeedbackQuestionRecipientAttributes.java

```diff
@@ -0,0 +1,89 @@
+package teammates.common.datatransfer;
+
+import java.util.Objects;
+
+/**
+ * The data transfer class for Recipient.
+ */
+public final class FeedbackQuestionRecipientAttributes {
+
+    private String name;
+    private String identifier;
+    private String section;
+    private String team;
+
+    private FeedbackQuestionRecipientAttributes(String name, String identifier) {
+        this.name = name;
+        this.identifier = identifier;
+        this.section = null;
+        this.team = null;
+    }
+
+    /**
+     * Returns a {@code FeedbackQuestionRecipientAttributes} with the recipient name and identifier.
+     */
+    public static FeedbackQuestionRecipientAttributes valueOf(String name, String identifier) {
```

Comment:
If you expose this method which passes to the constructor verbatim, you should just expose the constructor as public and discard this method

Reply:
Will update so, thanks for pointing it out!